datalad_catalog/catalog/README.md

```diff
@@ -0,0 +1,43 @@
+## The Catalog
+
+This is a browsable data catalog generated from metadata using [`datalad-catalog`](https://github.com/datalad/datalad-catalog/). It is a self-contained and static [VueJS](https://vuejs.org/)-based site that can be viewed in any modern internet browser.
+
+## Content
+
+The content of this root directory includes everything necessary to serve the static site:
+
+```
+.
+├── artwork
+├── assets
+├── config.json
+├── index.html
+├── metadata
+└── README.md
+```
+
+The `artwork` and `assets` directories contain images and web assets (such as JavaScript and CSS files) that support the rendering of the HTML components in `index.html`. The `config.json` file contains customizable configuration options supplied to `datalad-catalog` 
```

Comment:
", respectively (Such as JavaScript [...])"

Reply:
Misses a trailing full stop ('.') at the very end, once you're editing the line